### PR TITLE
Added bouncing ball with air friction model 

### PR DESCRIPTION
**Pull request by Aadesha**
Solved issue  #114 using TMJets.

### CODE REVIEW

File: models/bouncing ball with air friction/Hybrid_model_bouncing_ball.jl [hunk 1]
```diff
@@ -0,0 +1,81 @@
+using Dates; now()
+using Revise, MathematicalSystems,Reachability, LinearAlgebra, HybridSystems
+using LazySets; Interval(0, 1)
```
Comment (mforets) [inline models/bouncing ball with air friction/Hybrid_model_bouncing_ball.jl]:
```suggestion
using LazySets
```

File: models/bouncing ball with air friction/Hybrid_model_bouncing_ball.jl [hunk 2]
```diff
@@ -0,0 +1,81 @@
+using Dates; now()
+using Revise, MathematicalSystems,Reachability, LinearAlgebra, HybridSystems
```
Comment (mforets) [inline models/bouncing ball with air friction/Hybrid_model_bouncing_ball.jl]:
```suggestion
using Revise, MathematicalSystems, Reachability, LinearAlgebra, HybridSystems
```

File: models/bouncing ball with air friction/Hybrid_model_bouncing_ball.jl [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+@taylorize function bball_down!(t, x, dx)
+    dx[1] = x[2]
+    dx[2] = -9.8 + 0.1*(x[1])^2
+    return dx
+end
+
+function bouncing_ball()
+
+    automaton = LightAutomaton(2) # two nodes
```
Comment (mforets) [inline models/bouncing ball with air friction/Hybrid_model_bouncing_ball.jl]:
```suggestion
    automaton = LightAutomaton(2) # two modes
```

File: models/bouncing ball with air friction/Hybrid_model_bouncing_ball.jl [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+    inv_up = HPolyhedron([HalfSpace([-1.0, 0.0], 0.0),  # x >= 0
+                          HalfSpace([0.0, -1.0], 0.0)]) # v >= 0
+
+    inv_down = HPolyhedron([HalfSpace([-1.0, 0.0], 0.0),  # x >= 0
+                         HalfSpace([ 0.0, 1.0], 0.0)])  # v <= 0
+
+    m1 = ConstrainedBlackBoxContinuousSystem(bball_up!, 2, inv_up)
+
+    m2 = ConstrainedBlackBoxContinuousSystem(bball_down!,2, inv_down)
```
Comment (mforets) [inline models/bouncing ball with air friction/Hybrid_model_bouncing_ball.jl]:
```suggestion
    m2 = ConstrainedBlackBoxContinuousSystem(bball_down!, 2, inv_down)
```

File: models/bouncing ball with air friction/Hybrid_model_bouncing_ball.jl [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+
+
+    guard_alpha = HPolyhedron([HalfSpace([1.0, 0.0], 0.0),   #x>=0
+                              HalfSpace([-1.0, 0.0], 0.0)])  #x<=0
+    guard_beta =  HPolyhedron([HalfSpace([0.0, 1.0], 0.0),   #v<=0
+                               HalfSpace([0.0,-1.0], 0.0)])  #v>=0
+
+
+    A = [1.0 0.0; 0.0 -0.75]
```
Comment (mforets) [inline models/bouncing ball with air friction/Hybrid_model_bouncing_ball.jl]:
```suggestion
    A = [1.0 0.0; 0.0 -0.8]
```

File: models/bouncing ball with air friction/Hybrid_model_bouncing_ball.jl [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+    resetmaps = [t1,t2]
+
+    # switching
+    switching = AutonomousSwitching()
+    switchings = fill(switching, 2)
+
+    ℋ = HybridSystem(automaton, modes, resetmaps, switchings)
+
+    #initial condition in mode one
+    X0= Hyperrectangle(low=[ 4.9,-0.2], high=[ 5.1,0.0])
```
Comment (mforets) [inline models/bouncing ball with air friction/Hybrid_model_bouncing_ball.jl]:
```suggestion
    X0= Hyperrectangle(low=[4.9, -0.2], high=[5.1, 0.0])
```

File: models/bouncing ball with air friction/Hybrid_model_bouncing_ball.jl [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+
+    initial_condition = [(2,X0)]
```
Comment (mforets) [inline models/bouncing ball with air friction/Hybrid_model_bouncing_ball.jl]:
```suggestion
    initial_condition = [(2, X0)]  # initial condition in the "down" mode
```

**Comment by mforets** [comment]
Thanks for the contribution! Looks good to me. Would you mind pasting the plot here?

**Comment by schillic** [comment]
Closes #114.

**Pull request merged**